src/tests/Microsoft.PowerFx.Core.Tests.Shared/ExpressionTestHelpers/TestRunner.cs

```diff
@@ -136,7 +136,11 @@ public static string GetDefaultTestDir()
 
         public void AddDir(Dictionary<string, bool> setup, string directory = "")
         {
+#if NETCOREAPP3_1_OR_GREATER
+            // Can't define Shims on static classes
```

Comment:
Is this ok to not call on 462? 

We could also have a static class like FileShim and add helpers to that (that's still better than #ifdefs).